lib/fortnox/api/models/order.rb

```diff
@@ -19,6 +19,13 @@ class Order < Fortnox::API::Model::Base
 
         # OrderRows Separate object
         attribute :order_rows, Array[OrderRow]
+
+        # Override to_hash to convert nested OrderRows to hash correctly.
+        def to_hash *args
```

Comment:
I just gave it another try and we do need the `to_hash` method :( Without that you get something like this:

```
Fortnox::API::Model::Order.new(order_rows: Fortnox::API::Model::OrderRow.new).to_hash
# => {..., :order_rows=>[#<Fortnox::API::Model::OrderRow:0x007f0dc3fc77e0 @saved=false, @new=true, @account_number=nil, @article_number=nil, @contribution_percent=nil, @contribution_value=nil, @cost_center=nil, @delivered_quantity=nil, @description=nil, @discount=nil, @discount_type=nil, @house_work=nil, @house_work_hours_to_report=nil, @house_work_type=nil, @price=nil, @project=nil, @total=nil, @unit=nil, @vat=nil, @ordered_quantity=nil>]}
```